src/local/butler/reproduce.py

```diff
@@ -132,6 +134,24 @@ def _download_testcase(testcase_id, testcase, configuration):
   return testcase_path
 
 
+def _setup_x():
+  """Start Xvfb and blackbox before running the test application."""
+  environment.set_value('DISPLAY', ':99')
```

Comment:
It should check platform and bail out if not linux.

Reply:
Done